apps/astarte_trigger_engine/lib/astarte_trigger_engine/trigger.ex

```diff
@@ -0,0 +1,26 @@
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.TriggerEngine.KvStore do
+  use TypedEctoSchema
+
+  typed_schema "kv_store" do
```

Comment:
Ecto autogenerates an `id` field unless explicitly disabled

```suggestion
  @primary_key false
  typed_schema "kv_store" do
```